obspy/signal/tests/test_spectral_estimation.py

```diff
@@ -541,6 +542,25 @@ def callback(t_array):
             with np.errstate(under='ignore'):
                 fig.savefig(ic.name)
 
+    def test_earthquake_models(self):
+        """
+        Test earthquake models
+        """
+        ppsd = _get_ppsd()
+        fig = ppsd.plot(show_earthquakes=(1, 4, 5, 15), show=False)
+        ax = fig.axes[0]
+        for index, magnitude in enumerate([1.5, 2.5, 3.5]):
+            frequencies, power = earthquake_models[(magnitude, 10)]
+            power = np.array(power)
+            periods = 1 / np.array(frequencies)
+            power = power / (periods ** (-.5))
+            power = 20 * np.log10(power / 2)
+            line = ax.lines[index]
+            text = ax.tests[index]
+            assert all(line.get_ydata() == power)
+            assert all(line.get_xdata() == periods)
+            assert text.get_text() == native_str('$^{M3.5}_{100km}$')
```

Comment:
Sorry if my example snippet was misleading there, please use the `TestCase`s methods, like..

```python
self.assertEqual(line.get_ydata(), power))
self.assertEqual(line.get_xdata(), periods))
self.assertEqual(text.get_text(), native_str('$^{M3.5}_{100km}$'))
```

The string at the end has to be modified as well i think `'$^{M%.1f}_{10km}$' % magnitude`

Reply:
Done.